repo-scripts/api-documenter/src/documenters/MarkdownDocumenterHelpers.ts

```diff
@@ -168,6 +169,29 @@ function getHeadingAnchorForApiItem(apiItem: ApiItem): string {
   }
 }
 
+/**
+ * Generates a unique link for a function.  Example: "getArea_paramhashhere"
+ */
+function getFunctionOverloadAnchor(apiItem: ApiItem): string {
+  if (
+    ApiParameterListMixin.isBaseClassOf(apiItem) &&
+    apiItem.parameters.length > 0
+  ) {
+    // Create a sha256 hash from the parameter names and types.
+    const hash = createHash('sha256');
```

Comment:
I'm guessing we're going with the hash because it's really hard to keep track of which of the overloads is "first", "second", etc.? I seem to remember running into that problem last time. Just double checking. I think it's fine since they're generally pretty invisible except in the url bar.

Reply:
I went with hash because it was one of the simplest ways to get a deterministic link. Besides the challenges you ran into, numbering the overloads could change between publications based on re-ordering.

I considered including the param names in the links, but I felt like that could result in long link names and possibly more conflicts.